README.md

```diff
@@ -295,6 +295,37 @@ func main() {
 }
 ```
 
+## Emitting a schema.json
+
+It may be useful to emit a `schema.json` file containing the results of the
```

Comment:
Rewrite this sentence to start with why it's useful as opposed to the ominous "it may be useful"